source/dub/dependency.d

```diff
@@ -21,12 +21,16 @@ import std.exception;
 import std.string;
 import std.sumtype;
 
+struct PackageName {
+    string value;
+    alias value this;
```

Comment:
I feel dubious about whether we should use alias this here or not.

Reply:
Now that I've added toString member should we explicit use that and make value private and remove alias this?

Reply:
I'm keeping the alias this for now making the code smaller.